LEGO1/mxvideopresenter.cpp

```diff
@@ -2,8 +2,88 @@
 
 DECOMP_SIZE_ASSERT(MxVideoPresenter, 0x64);
 
+// OFFSET: LEGO1 0x1000c700 STUB
+void MxVideoPresenter::VTable0x5c()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c710 STUB
+void MxVideoPresenter::VTable0x60()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c720 STUB
+void MxVideoPresenter::VTable0x68()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c730 STUB
+void MxVideoPresenter::VTable0x70()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1000c740
```

Comment:
Can you mark this as `STUB`? Alternatively, maybe add a stub for the `Destroy` function at `100b27b0` and call it from the destructor, looks like it's the only thing it does anyway.